orchagent/aclorch.cpp

```diff
@@ -2903,8 +2903,16 @@ void AclOrch::doAclRuleTask(Consumer &consumer)
                 type = table_id == m_mirrorTableId[stage] ? ACL_TABLE_MIRROR : ACL_TABLE_MIRRORV6;
             }
 
-
-            newRule = AclRule::makeShared(type, this, m_mirrorOrch, m_dTelOrch, rule_id, table_id, t);
+            try
+            {
+                newRule = AclRule::makeShared(type, this, m_mirrorOrch, m_dTelOrch, rule_id, table_id, t);
+            }
+            catch (exception &e)
```

Comment:
>catch [](start = 12, length = 5)

The communication between orchagent and syncd on 201911 is async mode. Is it related to a syncd/SAI error code? In that case, will you catch anything?